lib/elixir/lib/inspect/algebra.ex

```diff
@@ -588,6 +588,7 @@ defmodule Inspect.Algebra do
     doc_cons(doc1, doc2)
   end
 
+  @doc false
```

Comment:
We should document it, given it is used by external modules. Apologies for missing this earlier.

Reply:
I gave it a shot but I'm not very confident [6fdd0b1](https://github.com/elixir-lang/elixir/pull/13893/commits/6fdd0b19f3a322174d9808862ca9fb10af96665a), feel free to edit and merge.